src/osd/PeeringState.h

```diff
@@ -61,7 +61,11 @@ struct PeeringCtx;
 
 // [primary only] content recovery state
 struct BufferedRecoveryMessages {
+  #if defined(WITH_SEASTAR)
```

Comment:
nit: I believe our convention recommends to put such macros at the 0th indent level.

Reply:
Thanks for the heads up, fixed it.